.travis.yml

```diff
@@ -1,16 +1,15 @@
 sudo: require
 language: cpp
-dist: trusty
+dist: xenial
```

Comment:
Just for curiosity, what is the advantage of using xenial?

I think we should move the dist inside the matrix, under the os.
```
matrix:
  include:
    - os: linux
      dist: trusty
    - os: osx
      osx_image: xcode7.2
```

Reply:
Well, first of all, I wanted to see if it would work because I had a problem with cmake 3.9. So I wanted to run it with 3.10 and install it but then I saw they had xenial and thought I'd try that first.

But I haven't figured out what was exactly wrong for 3.9 (I think it should work in principle). So we should think about what to do with this. The easiest would be to just bump cmake required version to 3.10. I'm not convinced what the right thing to do here is.